docs/DeveloperGuide.md

```diff
@@ -1969,32 +1969,70 @@ Value of the input command changes from "redit cl/Jayden a/Blk" to "redit cl/Jay
 
 #### Sort all _clients_ by name
 
-1. Sorting the entire True Rental's database by name.
+Sorting the entire True Rental's database by name.
 
-    1. Prerequisites: List all _clients_ using the `list` command. Multiple _clients_ in the list.
+<box type="warning">
+
+**Prerequisites**: List all _clients_ using the `list` command. Multiple _clients_ in the list.
 
-    2. Test case: `sort`<br>
-       Expected: Client list will be sorted in alphabetical order by name. A command success message for name will be
-       displayed in the result display box.
+</box>
+
+<box type="info" seamless>
+
+**Test case 1**: `sort`
+
+<box type="success">
+
+Client list will be sorted in alphabetical order by name. A command success message for name will be
+displayed in the result display box.
+
+</box>
+
+</box>
 
 #### Clear
 
-1. Clearing the entire True Rental's database, wiping all clients and their respective rental information.
+Clearing the entire True Rental's database, wiping all clients and their respective rental information.
+
+<box type="warning">
+
+**Prerequisites**: There is at least 1 client in the application.
```

Comment:
Just checked on the `clear` command, it actually does not need any client to be able to run (when no data), it just clear the data regardless. You may want to remove prerequisites and rephrase the description to xxx (if any) or anything make sense.

Reply:
Good point, have updated on this!